mozilla/foundation.mozilla.org
# added airtable to blog pages

Closes #7833 
Related PRs/issues #

**Link to sample test page**: https://foundation-s-airtable-o-xvknqp.mofostaging.net/en/blog/initial-test-blog-post-with-fixed-title/ 

**Steps to test:** 

1. Please visit the test page above
2. Scroll down and view the new embedded airtable component. 
3. If everything looks as expected, testing is complete! 


## Checklist
**Changes in Models:**
